_drafts/shallow-clone-post-mortem.md

```diff
@@ -0,0 +1,56 @@
+[Recently](https://github.com/CocoaPods/CocoaPods/issues/4989) we were made aware of issues that our users were having
+cloning the [CocoaPods OSS spec-repo](https://github.com/CocoaPods/Specs).
```

Comment:
Maybe we could make it even more explicit then? :) 

`… cloning the CocoaPods Master spec-repo, which contains specifications for projects shared with the community.`

That way we would also emphasise that the repo does not contains the libraries itself, which was a common opportunity to be wrong on the internet previously.